verilog/tools/ls/verible-verilog-ls_test.sh

```diff
@@ -43,38 +55,17 @@ MSG_OUT=${TEST_TMPDIR:-/tmp/}/test-lsp-out-msg.txt
 awk '/^{/ { printf("Content-Length: %d\r\n\r\n%s", length($0), $0)}' > ${TMP_IN} <<EOF
 {"jsonrpc":"2.0", "id":1, "method":"initialize","params":null}
 
-# Testing a file with syntax errors: this should output some diagnostic
-{"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file://syntaxerror.sv","text":"brokenfile\n"}}}
+# Testing a file that should not create diagnostics
+{"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file://simple.sv","text":"module simple();\nendmodule\n"}}}
 
 # Let's manually request these diagnostics
-{"jsonrpc":"2.0", "id":2, "method":"textDocument/diagnostic","params":{"textDocument":{"uri":"file://syntaxerror.sv"}}}
+{"jsonrpc":"2.0", "id":2, "method":"textDocument/documentSymbol","params":{"textDocument":{"uri":"file://simple.sv"}}}
 
-# A file with a lint error (no newline at EOF). Then editing it and watching diagnostic go away.
-{"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file://mini.sv","text":"module mini();\nendmodule"}}}
-
-# Requesting a code-action exactly at the position the EOF message is reported.
-# This is an interesting special case, as the missing EOF-newline is an empty
-# range, yet it should be detected as overlapping with that diagnostic message.
-{"jsonrpc":"2.0", "id":10, "method":"textDocument/codeAction","params":{"textDocument":{"uri":"file://mini.sv"},"range":{"start":{"line":1,"character":9},"end":{"line":1,"character":9}}}}
-{"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument":{"uri":"file://mini.sv"},"contentChanges":[{"range":{"start":{"character":9,"line":1},"end":{"character":9,"line":1}},"text":"\n"}]}}
-{"jsonrpc":"2.0", "id":11, "method":"textDocument/documentSymbol","params":{"textDocument":{"uri":"file://mini.sv"}}}
+# Close the file
 {"jsonrpc":"2.0","method":"textDocument/didClose","params":{"textDocument":{"uri":"file://mini.sv"}}}
```

Comment:
According to the description above we should do `didOpen`, `documentSymbol`, `didClose` and `documentSymbol` on the same file. Currently, we open and request symbol for `simple.sv` file, while closing and re-requesting symbol for `mini.sv`. I guess we should use the same file name for all requests.

Reply:
Oh, indeed. Good find, thanks for the thorough review.
Fixed.